src/test/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/AdditionalParameterRegExTest.java

```diff
@@ -12,65 +15,65 @@
   public void testSingleParameter() throws Exception {
     AbstractMap.SimpleEntry<String, String> test1 =
         StashRepository.getParameter("p:customer=Nickesocke");
-    assert (test1 != null);
-    assert ("customer".equals(test1.getKey()));
-    assert ("Nickesocke".equals(test1.getValue()));
+    assertThat(test1, is(notNullValue()));
+    assertThat(test1.getKey(), is("customer"));
+    assertThat(test1.getValue(), is("Nickesocke"));
   }
 
   @Test
   public void testSpacedParameter() throws Exception {
     AbstractMap.SimpleEntry<String, String> test1 =
         StashRepository.getParameter("p:customer=Nicke socke#");
-    assert (test1 != null);
-    assert ("customer".equals(test1.getKey()));
-    assert ("Nicke socke#".equals(test1.getValue()));
+    assertThat(test1, is(notNullValue()));
+    assertThat(test1.getKey(), is("customer"));
+    assertThat(test1.getValue(), is("Nicke socke#"));
   }
 
   @Test
   public void testBlankParameter() throws Exception {
     AbstractMap.SimpleEntry<String, String> test1 =
         StashRepository.getParameter("p:the_blank_parameteR=");
-    assert (test1 != null);
-    assert ("the_blank_parameteR".equals(test1.getKey()));
-    assert ("".equals(test1.getValue()));
+    assertThat(test1, is(notNullValue()));
+    assertThat(test1.getKey(), is("the_blank_parameteR"));
+    assertThat(test1.getValue(), isEmptyString());
   }
 
   @Test
   public void testInvalidParameter() throws Exception {
     AbstractMap.SimpleEntry<String, String> test1 =
         StashRepository.getParameter("p:if apa=Nickesocke");
-    assert (test1 == null);
+    assertThat(test1, is(nullValue()));
     AbstractMap.SimpleEntry<String, String> test2 =
         StashRepository.getParameter("p:apa==Nickesocke");
-    assert (test2 != null
-        && test2.getKey().equals("apa")
-        && test2.getValue().equals("=Nickesocke"));
+    assertThat(test2, is(notNullValue()));
+    assertThat(test2.getKey(), is("apa"));
+    assertThat(test2.getValue(), is("=Nickesocke"));
     AbstractMap.SimpleEntry<String, String> test3 =
         StashRepository.getParameter(
             "p:I want to make sure that a use of = will not trigger parameter");
-    assert (test3 == null);
+    assertThat(test3, is(nullValue()));
     AbstractMap.SimpleEntry<String, String> test4 = StashRepository.getParameter("p:=nothing");
-    assert (test4 == null);
+    assertThat(test4, is(nullValue()));
     AbstractMap.SimpleEntry<String, String> test5 = StashRepository.getParameter("=nothing");
```

Comment:
Can you inline the local variables? `test5` is an awful name

Reply:
I agree about awful names. I just used `entry` everywhere. I'm not sure about inlining, it would mix test setup and test verification into the same expression. I can do it is you insist, but I'd rather not.

Reply:
My point is: if you can't give the variable a good name, don't extract it. `Entry` doesn't bring any more information (it is a bit better than `test5`).

```
    assertThat(
            StashRepository.getParameter("=nothing"),
            is(nullValue())
   );
```
I think above is easier to read, as you don't need to keep the extra `entry` in your head. There is a also the in-elegance of re-assigning the local variable. I would prefer local variables to be final.

It's not a big problem though, the tests still look better overall after this change

Reply:
I've inlined the variables wherever practical. `singleParameters` and `multipleParameters` need multiple checks, so they remain named variables.

Reply:
Thanks
The previous version was good enough for me, I'm just waiting for possible input from @jimklimov